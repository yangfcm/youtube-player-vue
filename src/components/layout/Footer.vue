<template>
  <div class="ui footer segment">
    <div class="ui center aligned container">
      My Youtube player developed by Fan Y.
    </div>
    <div class="ui center aligned container">
      Powered by <a href="https://vuejs.org/index.html" target="__blank">Vue</a>
    </div>
  </div>
</template>

<script>
export default {};
</script>

<style scoped></style>
