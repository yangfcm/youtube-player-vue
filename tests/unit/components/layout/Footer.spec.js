import { shallowMount } from "@vue/test-utils";
import Footer from "@/components/layout/Footer.vue";

const wrapper = shallowMount(Footer);
test("test footer", () => {
  expect(wrapper).toMatchSnapshot();
});
