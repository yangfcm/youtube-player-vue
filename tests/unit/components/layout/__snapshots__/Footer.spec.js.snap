// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test footer 1`] = `
<div class="ui footer segment">
  <div class="ui center aligned container">
    My Youtube player developed by Fan Y.
  </div>
  <div class="ui center aligned container">
    Powered by <a href="https://vuejs.org/index.html" target="__blank">Vue</a></div>
</div>
`;
